library/src/main/java/org/apache/apex/malhar/lib/state/managed/Bucket.java

```diff
@@ -232,6 +232,12 @@ public int hashCode()
 
     protected ConcurrentLinkedQueue<Long> windowsForFreeMemory = new ConcurrentLinkedQueue<>();
 
+    private static boolean disableBloomFilterByDefault = false;
+    private boolean disableBloomFilter = disableBloomFilterByDefault;
+    private static int bloomFilterDefaultBitSize = 1000000;
```

Comment:
I think the properties which allow to configure a bloom filter should not be per bucket. I am talking about variables like: `disableBloomFilter` and `bloomFilterBitSize`. I think these should be kept configurable. Would the user ever want to configure these differently for different buckets? How would a user of managed state be able to configure the bloom filter size?